tt.sh

```diff
@@ -35,13 +35,24 @@ function time_in {
 		time="$(date '+%Y-%m-%d') ${1}:00"
 		activity="${@:2}"
 	fi
-	echo "Begin ${activity} at ${time}"
-	echo "i ${time} ${activity}" >> $TIME_FILE
+	#
+	# Split the activity into:
+	#
+	#   - the project: anything before the first whitespace(s)
+	#     whitespace-containing project name aren't supported.
+	#     (maps to the hledger transaction virtual account)
+	#
+	#   - the project task: anything following the first whitespace(s)
+	#     (maps to the hledger transaction description)
+	#
+	local project_task=$(echo "${activity}" | sed -E "s/[[:blank:]]+/  /")
```

Comment:
How would you feel about separating the project and task with a *double* space? That would allow spaces in project names, and follow the typical ledger/hledger convention for separating accounts and amounts in journal files.